### PR TITLE
Added ConsoleUi class to project + added showMultipleChoice method

### PR DESCRIPTION
**Pull request by CodingMarco**

### CODE REVIEW

**Comment by Bennik2000** [comment]
Looks good!

File: FourInARow/FourInARowLib/ConsoleUi.h [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include "Ui.h"
+
+class ConsoleUi : public Ui
```
Comment (Bennik2000) [inline FourInARow/FourInARowLib/ConsoleUi.h:4]: There could be a simple comment which describes what the class does. 
However we have to discuss if and how many comments we want to add to this project

Comment (CodingMarco) [reply FourInARow/FourInARowLib/ConsoleUi.h:4]: Just for fun - https://youtu.be/-AQfQFcXac8?t=148 ;)

Comment (Bennik2000) [reply FourInARow/FourInARowLib/ConsoleUi.h:4]: I know this video and it is somewhat true ;)  
IMHO code should be written that way that no comments are needed. 
http://nicolecarpenter.github.io/2016/03/08/clean-code-chapter-4-comments.html

**Pull request merged**